.env.example

```diff
@@ -4,6 +4,7 @@
 #
 MYUSA_KEY: consumer_public_key
 MYUSA_SECRET: consumer_secret_key
+DEFAULT_URL_HOST: your_local_server_address
```

Comment:
we do need this for urls to work in emails -- see `config/initializers/mailer.rb` -- this is a newish change